ow_plexil/src/plans/FaultDependenciesModel.xml

```diff
@@ -0,0 +1,95 @@
+<System>
+  <Subsystems>
+
+    <Subsystem Name="Power">
+      <Faults>
+        <Fault Name="LOW_STATE_OF_CHARGE">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="INSTANTANEOUS_CAPACITY_LOSS">
+        </Fault>
+        <Fault Name="THERMAL_FAULT">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+      </Faults>
```

Comment:
In my understanding, an instantaneous capacity loss fault is a momentary, transient fault that doesn't necessarily impact anything by itself. @chetankul could confirm.

Reply:
An impacts clause causes the target subsystem or procedure to become inoperable when that fault is triggered.

If like Mike mentioned, the instantaneous capacity loss fault is momentary we wouldnt want to necessarily cause the whole power subsystem to be inoperable. 

A fault within a subsystem does not necessarily need to make its parent inoperable when triggered. But it will make its parent subsystem faulty.

Reply:
Thanks, that clarifies things